### PR TITLE
Fix loss keys of named class in `Html::addCssClass()` when class in passed options is string

### PR DESCRIPTION
**Pull request by vjik**
| Q             | A
| ------------- | ---
| Is bugfix?    | ✔️
| New feature?  | ❌
| Breaks BC?    | ❌
| Fixed issues  |

### CODE REVIEW

File: tests/common/HtmlTest.php [hunk 1]
```diff
@@ -882,12 +882,6 @@ public function testAddCssClass(): void
         Html::addCssClass($options, ['test3']);
         $this->assertSame(['class' => ['test', 'test2', 'test3']], $options);
 
-        $options = [
-            'class' => 'test',
-        ];
-        Html::addCssClass($options, ['test1', 'test2']);
-        $this->assertSame(['class' => 'test test1 test2'], $options);
```
Comment (Tigrov) [inline tests/common/HtmlTest.php:889]: Maybe leave the test with new result?
`$this->assertSame(['class' => ['test', 'test1', 'test2']], $options);`

Comment (vjik) [reply tests/common/HtmlTest.php:889]: Added as separate test: https://github.com/yiisoft/html/pull/171/files#diff-2bb921927af1a632486e414d8a3f66c516f7fdde7b122a677eb2f4183c0d6e27R908

**Review by Tigrov — APPROVED** [review]
👍

File: CHANGELOG.md [hunk 2]
```diff
@@ -2,7 +2,7 @@
 
 ## 3.1.1 under development
 
-- no changes in this release.
+- Bug #171: Fix loss keys of named class in `Html::addCssClass()` when class in passed options is string (@vjik)
```
Comment (arogachev) [inline CHANGELOG.md:5]:
```suggestion
- Bug #171: Fix loss of keys for named class in `Html::addCssClass()` when class in passed options is a string (@vjik)
```

**Review by arogachev — APPROVED** [review]

**Pull request merged**